tasks/packagemgmt.yml

```diff
@@ -205,9 +205,11 @@
 
 - name: dnf upgrade
   become: 'yes'
-  command: dnf --assumeyes upgrade-minimal
-  register: dnf_upgrade_response
-  changed_when: dnf_upgrade_response.stdout.find('Nothing to do.') == -1
```

Comment:
keep the `changed_when:` (maybe need a rewrite when switching to `dnf`) so the task only reports as changed when there's actually a change.

Reply:
I think that it is not necessary here, because when dnf module provide any changes, then we get correctly yellow status:

```
TASK [dnf upgrade] ***********
changed: [172.17.88.251]
```

Then next run gives the green result:

```
TASK [dnf upgrade] *******
ok: [172.17.88.251]
```

By other words, the dnf module handle the changed stated correctly itself...

But `Nothing To Do.` check will still works too, but id i do not miss something, it's obsolete